docs/formatting-and-static-code-checks.md

```diff
@@ -0,0 +1,48 @@
+In order to make it maximally easy to work on spamgourmet we are using static
+code checking and opinionated. The primary tool of choice for this is
+`pre-commit` which allows us to run all of the static checks both just before
+committing in git, manually over all the files on demand and also in our
+automate builds system (currently GitHub actions)
+
+Line Width handling
+###################
+
+Our agreed policy on line widths is approximately:
+
+- aim for lines less than 80 characters wide in English text
+  (e.g. documentation)
+- aim for lines less than 100 characters wide in code
```

Comment:
Can we just combine the first two points into "aim for lines less than 80 characters"? It seems soft enough that people can exceed it by their own judgment.